.github/workflows/ci.yml

```diff
@@ -171,3 +171,36 @@ jobs:
         run: |
           docker run -u $(id -u) --rm -v ${{ github.workspace}}:/awx_devel/:Z \
             --workdir=/awx_devel ghcr.io/${{ github.repository_owner }}/awx_devel:${{ github.base_ref }} make ui-test
+
+  e2e-test:
```

Comment:
Unless we want this to run on every PR, I think this should be moved into its own workflow rather than get added here.

Reply:
Ok, I think the idea is run on every PR right? Any reason why we shouldn't run for every PR?

Reply:
Ok, moved this to another workflow and will run when the label `state:needs_test` is applied to the PR.